src/views/project/institutions/CreateView.vue

```diff
@@ -34,6 +34,22 @@ async function addInstitution(institutionId, index) {
   }
 }
 
+async function buildInstitution() {
+  const url = `${
+    import.meta.env.VITE_API_URL
+  }/projects/${projectId}/institutions/build`
+  const response = await axios.post(url, { name: searchTerm.value })
+
+  if (response.status == 200) {
+    const institution = response.data.newInstitution
+    searchList.value.push(institution)
+
+    addInstitution(institution.institution_id, searchList.value.length - 1)
+  } else {
+    alert('Could not add institution to the database')
+  }
+}
```

Comment:
All actions should follow the ACID (e.g. Atomicity, Consistency, Isolation, and Durability). Either the action succeeds or fails - it should never partially succeed because that results inconsistent data.

Here are the two options: 
1. Re-use "addInstitution" which will accepts an `institution_id` and `name` . If `name` is specified, look up the name in the database to ensure that it doesn't exist if so, return the institution_id. Otherwise create institution. Then add the institution to the project.
2. Create a new endpoint, createInstitution which will create the insitutiion and add it to the project. If the institution already exists throw an error.